front/src/common/redux/auth/auth.api.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+const url = 'http://127.0.0.1:5000/auth/login';
```

Comment:
Please, read how to use environment variables in react like REACT_URL
https://create-react-app.dev/docs/adding-custom-environment-variables/

Reply:
done